code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -423,7 +423,7 @@
 	fdel(json_file)
 	WRITE_FILE(json_file, json_encode(file_data))
 
-/mob/living/simple_animal/pet/dog/corgi/Ian/Life()
+/mob/living/simple_animal/pet/dog/Life()
```

Comment:
Put this with the other dog procs

Reply:
@Firecage just move this proc please.

Reply:
@AnturK Heh, I would have done so earlier. Rounds on Sybil 2 kiiiinda lasts a long time though.